tests/meson.build

```diff
@@ -0,0 +1,78 @@
+catch2 = dependency('catch2')
```

Comment:
Just for my understanding, this will download catch only if not found? And any version can be found, but 2.13.7 will be downloaded, correct?

Reply:
Yes. Based on the setting of `--wrap-mode=default` (you can set it to `=forcefallback` if you'd prefer to download it no matter what) it will:
- try to find any version using pkg-config
- try to find any version using cmake (by running a crafted CMakeLists.txt, seeing if find_package() works, and extracting that info using the trace parser)
- download the pinned version available as a meson wrap